src/myframe.h

```diff
@@ -26,9 +26,21 @@ struct MyFrame : wxFrame {
     ColorDropdown *textdd = nullptr;
     ColorDropdown *borddd = nullptr;
     
-    wxString GetPath(const wxString &relpath) {
-        if (!exepath_.Length()) return relpath;
-        return exepath_ + "/" + relpath;
+    wxString GetDocPath(const wxString &relpath) {
+        #ifdef TREESHEETS_DOCDIR
+            return TREESHEETS_DOCDIR "/" + relpath;
+        #else
+            if (!exepath_.Length()) return relpath;

```

Comment:
I'd vote for always trying this first, rather than compile-time.

I'd think of the user that has an installed version, and also made a local build in their homedir to try a new feature. Likely the local build should take precedence.

Reply:
Done.